testsuite/model/src/test/java/org/keycloak/testsuite/model/KeycloakModelTest.java

```diff
@@ -24,6 +24,7 @@
 import org.keycloak.authorization.DefaultAuthorizationProviderFactory;
 import org.keycloak.authorization.store.StoreFactorySpi;
 import org.keycloak.cluster.ClusterSpi;
+import org.keycloak.common.util.Time;
```

Comment:
unused import?

Reply:
It seems I did some mistake during rebase. The changes in the model tests should not be there. I will fix this